internal/envoy/bootstrap.go

```diff
@@ -102,6 +105,20 @@ func (c *BootstrapConfig) GetAdminAccessLogPath() string {
 func (c *BootstrapConfig) GetDNSLookupFamily() string {
 	return stringOrDefault(c.DNSLookupFamily, "auto")
 }
+
+// ValidAdminAddress checks if the address supplied is
+// "localhost" or an IP address. Only a Unix Socket
+// is supported for this address to mitigate security.
+func ValidAdminAddress(address string) error {
+	// Value of "localhost" is invalid.
+	if address == "localhost" ||
+		net.ParseIP(address) != nil ||
+		!strings.Contains(address, "/") {
+		return fmt.Errorf("invalid value %q, must be a unix socket name", address)
```

Comment:
maybe worth separating the error out for the localhost and ip address cases to say explicitly they are not allowed